README.md

```diff
@@ -6,7 +6,7 @@ Communication library used by SYSL-generated code written in Go.
 
 Go get the repository
 
-    go get github.com/anz-bank/sysl-go-comms
+    go get github.com/anz-bank/sysl-go
```

Comment:
The first line of README needs change as well.
```diff
- # sysl-go-comms
+ # sysl-go
```

Reply:
There're several *.md still contain "sysl-go-comms". You can search "sysl-go-comms" through the whole project and replace them:)

Reply:
Thanks, fixed in 682719e11aad09f76074240403f24c9eaa220025